corehq/apps/app_manager/forms.py

```diff
@@ -63,3 +64,76 @@ def clean(self):
         if self.cleaned_data.get('linked'):
             if not LINKED_APPS.enabled(domain):
                 raise forms.ValidationError("The target project space does not have linked apps enabled.")
+
+
+class PromptUpdateSettingsForm(forms.Form):
```

Comment:
Looking at the UI on staging the save button is in a different place from the rest of app manager. It's also a different color and doesn't change state when a data change is made.